### PR TITLE
Drop support for Python 2.6 and 3.2, claim support for Python 3.5.

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 =========
 
-4.2.3 (unreleased)
+4.3.0 (unreleased)
 ------------------
 
-- TBD
+- Drop support for Python 2.6 and 3.2.
+
+- Claim support for Python 3.5.
```
Comment (tseaver) [inline CHANGES.rst]: Prefer "Add support" to "Claim support":  there *isn't* support until we test for it.

File: .coveragerc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[run]
+branch = True
+source = AccessControl
+
+[report]
+precision = 2
+omit = */tests.py
```
Comment (tseaver) [inline .coveragerc]: Can you add `show_missing = True` to the `[report]` section, too?  We are missing two lines at present.

Comment (tseaver) [inline .coveragerc]: Also, why omit reporting on the tests?  Those should be at 100% too.

File: .coveragerc [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[run]
+branch = True
+source = AccessControl
```
Comment (tseaver) [inline .coveragerc]: And this line can't be correct:  this is `Acquisition`, not `AccessControl`.

Comment (icemac) [inline .coveragerc]: Good catch, I copied the file from there and this line seem to have slipped through my fingers. But it seems that `AccessControl` here does not harm. 
Changed it anyway.

**Comment by icemac** [comment]
@tseaver Thank you for your feedback, I changed the code accordingly.

**Pull request merged**